forEach/challenges-01.test.js

```diff
@@ -12,6 +12,8 @@
 const pushIntoSecond = (arr) => {
   const second = [];
   // Solution code here...
+  arr.forEach((el) => second.push(el));
```

Comment:
please be careful when doing assignments to read the instructions. The instructions state to use a for loop for this question.